DbAdapterTestUtilities/Comparators/EntityComparator.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include "EntityComparatorConfiguration.h"
+
+#include <gtest/gtest.h>
+
+namespace systelab::db::test_utility {
+
+	class EntityComparator
```

Comment:
Are you using those classes? I though you moved to specific comparators for each type like you have below

Reply:
I did, but in order to implement the comparators these classes were needed

Reply:
In any case, I'll have a look at it, cause giving a second read at the code, it seems that maybe those files could be unneccesary